iroh-net/src/lib.rs

```diff
@@ -4,6 +4,39 @@
 //! interface to [QUIC] connections and streams to the user, while implementing direct
 //! connectivity using [hole punching] complemented by relay servers under the hood.
 //!
+//! Connecting to a remote node looks roughly like this:
+//!
+//! ```no_run
+//! # use iroh_net::{Endpoint, NodeAddr};
+//! # async fn wrapper() -> testresult::TestResult {
+//! let addr: NodeAddr = todo!();
+//! let ep = Endpoint::builder().bind().await?;
+//! let conn = ep.connect(addr, b"my-alpn").await?;
+//! let mut send_stream = conn.open_uni().await?;
+//! send_stream.write_all(b"msg").await?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! Accepting connections is a little more work:
```

Comment:
```suggestion
//! The other side can accept incoming connections like this:
```

I'm... not a huge fan of talking about how much work sth is :D
This is great! No need to downplay iroh.

Reply:
good call